docusaurus/docs-internals/editor-controls/multivalue-controls/iconSetter.md

```diff
@@ -0,0 +1,182 @@
+| `id`                | `string`                                 |    -    | The identifier of the key where the iconSetter will save your data                                                                                                                                                                |
+| `type`              | `string`                                 |    -    | Type should be `"iconSetter"` to use this control                                                                                                                                                                                 |
+| `label?`            | `string`                                 |    -    | The label displayed on the left side of the control                                                                                                                                                                               |
+| `position?`         | `number`                                 |    -    | The position of the control in toolbar                                                                                                                                                                                            |
+| `devices?`          | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`         | `boolean`                                | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
+| `value.name?`       | `string`                                 |    -    | The name of the icon being set.                                                                                                                                                                                                   |
+| `value.type?`       | `string`                                 |    -    | The type of the icon being set.                                                                                                                                                                                                   |
+| `value.filename?`   | `string`                                 |    -    | The filename of the icon being set.                                                                                                                                                                                               |
```

Comment:
these params is missing in props

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/5f7247b5-850d-4c75-a39a-4cb4f67fe031)